code/game/machinery/vending.dm

```diff
@@ -880,12 +880,11 @@
 	desc = "If you want to get cancer, might as well do it in style"
 	product_slogans = "Space cigs taste good like a cigarette should.;I'd rather toolbox than switch.;Smoke!;Don't believe the reports - smoke today!"
 	product_ads = "Probably not bad for you!;Don't believe the scientists!;It's good for you!;Don't quit, buy more!;Smoke!;Nicotine heaven.;Best cigarettes since 2150.;Award-winning cigs."
-	vend_delay = 34

```

Comment:
Only machine in the entire list to use this variable for some reason. It doesn't even seem to have any reason to vend slower, so I just canned it out. Impossible to tell the difference